e3nn/spherical_tensor.py

```diff
@@ -79,7 +81,11 @@ def spherical_harmonics_dirac(lmax, alpha, beta):
     """
     ls = list(range(lmax + 1))
     a = sum(2 * l + 1 for l in ls) / (4 * math.pi)
-    return rsh.spherical_harmonics_alpha_beta(ls, torch.tensor(alpha), torch.tensor(beta)) / a
+    if "torch" not in str(type(alpha)):
```

Comment:
There's probably a more elegant way of doing this.

Reply:
Personally I would impose the input to be a tensor but ok

Reply:
Your test uses floats. That's why I add the conversion. Do you want me to change your test to tensors?

Reply:
yes let change the test